slave/Example.md

```diff
@@ -27,11 +27,10 @@ func main() {
 	flag.Parse()
 
 	server := &http.Server{
-		Addr:    addr,
 		Handler: new(SleepyHandler),
 	}
 
-	slave.Serve(server, 5 * time.Minute)
+	slave.ServeHTTP(server, addr, 5 * time.Minute)
```

Comment:
`ServeHTTP(ResponseWriter, *Request)` is used by the `http.Handler` interface. How about `slave.ListenAndServeHTTP` instead?

Reply:
👍